js/models/todo.js

```diff
@@ -1,22 +1,32 @@
-App.Todo = DS.Model.extend({
-  title: DS.attr('string'),
-  isCompleted: DS.attr('boolean')
+Abs.Tasks = DS.Model.extend({
```

Comment:
I renamed it because in my opinion the word "Tasks" is more clear than "Todo" 


     On Monday, September 28, 2015 6:01 PM, Ruan Naude <notifications@github.com> wrote:
   

 In js/models/todo.js:> @@ -1,22 +1,32 @@
> -App.Todo = DS.Model.extend({
> -  title: DS.attr('string'),
> -  isCompleted: DS.attr('boolean')
> +Abs.Tasks = DS.Model.extend({
Why rename it from Todo to Tasks?—
Reply to this email directly or view it on GitHub.